.travis.yml

```
@@ -9,7 +9,7 @@ jobs:
       name: "Docker"
       before_install:
         - docker pull radiorabe/suisa_sendemeldung
-      script: docker build -t suisa_sendemeldung --cache-from radiorabe/suisa_sendemeldung .
+      script: docker build -t suisa_sendemeldung --cache-from radiorabe/suisa_sendemeldung . && docker run --rm -ti suisa_sendemeldung -h
```

You could also split the script onto two lines instead of using `&&`.
```suggestion
      script:
        - docker build -t suisa_sendemeldung --cache-from radiorabe/suisa_sendemeldung .
        - docker run --rm -ti suisa_sendemeldung -h
```

---

That was fast! I just wanted to ask you about a cleaner way. :+1: